code/data_structures/heap/min_heap/min_heap.c

```diff
@@ -0,0 +1,129 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+//gives the parent value.
+int parent(int i){
+	int p;
```

Comment:
If you want, you could eliminate the variable, and directly return the result.
I would simplify this function to:

```C
return (i%3==) ? (i/3-1) : (i/3);
```